pkg/pullsync/pb/pullsync.proto

```diff
@@ -11,7 +11,8 @@ option go_package = "pb";
 message Syn {}
 
 message Ack {
-  repeated uint64 Cursors = 1;
+  uint64 Epoch = 1;
```

Comment:
If you mark `Epoch` as `2`, protobuf takes care of backwards compatibility and you don't need to change the `protocolVersion` below (which is more for backwards incompatible logical changes).